SpookyStation fixes and edits

BlueWildrose opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

- Bartholomew says the correct year
- Telecomms is fixed.
- Adds colormates to the changing rooms

## Why It's Good For The Game

Was just in time for 2023's ball. Feel free to merge.

## Changelog
:cl:
tweak: Citadel Halloween Ball 2023
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

BlueWildrose committed:
spooky 2023

BlueWildrose committed:
Spooky colormates, better telecomms?

SandPoot commented on code/modules/holiday/halloween/bartholomew.dm:
```diff
@@ -14,7 +14,7 @@
 /obj/item/barthpot/attackby(obj/item/I, mob/user, params)
 	if(BallTutorial)
-		say("Hello and welcome to the annual Citadelstation Spookyball 2021! CENTCOM requisitioned this old hospital as a new colony site two years ago, and we’re back again after a few additions here and there. Next to me you can find all the tools you’ll need to build a nice private house, if you’re here for that kind of thing. The axes will chop trees, and give you wood. Shovels will remove grass an ...
+		say("Hello and welcome to the annual Citadel Station Spooky Ball 2023! CENTCOM requisitioned this old hospital as a new colony site four years ago, and we’re back again after a few additions here and there. Next to me you can find all the tools you’ll need to build a nice private house, if you’re here for that kind of thing. The axes will chop trees, and give you wood. Shovels will remove grass ...
```
```suggestion
		say("Hello and welcome to the annual Citadel Station Spooky Ball [time2text(world.realtime, "YYYY")]! CENTCOM requisitioned this old hospital as a new colony site four years ago, and we’re back again after a few additions here and there. Next to me you can find all the tools you’ll need to build a nice private house, if you’re here for that kind of thing. The axes will chop trees, and give you wood. Shovels will remove grass and things, which you can use the grass to make beds and backets. The pickaxe will allow you pick out the nearby rocks, if you’re more interested in building a cave dwelling. Finally the umbrella and light sources are because it’s spooky, and it might rain! As for interesting spots; There’s the old abandoned mansion which you can get to by going through the entrance and towards the east. There’s also a mech arena directly south, and a racetrack in the caves to the right of the arena, follow the lanterns. And make sure to explore the caves too! Lots of neat things to find!")
```
future proofing :)

SandPoot commented on code/modules/holiday/halloween/bartholomew.dm:
```diff
@@ -24,23 +24,23 @@
 /obj/item/barthpot/on_attack_hand(mob/user, act_intent = user.a_intent, unarmed_attack_flags)
 	if(BallTutorial)
-		say("Hello and welcome to the annual Citadelstation Spookyball 2021! CENTCOM requisitioned this old hospital as a new colony site two years ago, and we’re back again after a few additions here and there. Next to me you can find all the tools you’ll need to build a nice private house, if you’re here for that kind of thing. The axes will chop trees, and give you wood. Shovels will remove grass an ...
+		say("Hello and welcome to the annual Citadel Station Spooky Ball 2023! CENTCOM requisitioned this old hospital as a new colony site four years ago, and we’re back again after a few additions here and there. Next to me you can find all the tools you’ll need to build a nice private house, if you’re here for that kind of thing. The axes will chop trees, and give you wood. Shovels will remove grass ...
```
```suggestion
		say("Hello and welcome to the annual Citadel Station Spooky Ball [time2text(world.realtime, "YYYY")]! CENTCOM requisitioned this old hospital as a new colony site four years ago, and we’re back again after a few additions here and there. Next to me you can find all the tools you’ll need to build a nice private house, if you’re here for that kind of thing. The axes will chop trees, and give you wood. Shovels will remove grass and things, which you can use the grass to make beds and backets. The pickaxe will allow you pick out the nearby rocks, if you’re more interested in building a cave dwelling. Finally the umbrella and light sources are because it’s spooky, and it might rain! As for interesting spots; There’s the old abandoned mansion which you can get to by going through the entrance and towards the east. There’s also a mech arena directly south, and a racetrack in the caves to the right of the arena, follow the lanterns. And make sure to explore the caves too! Lots of neat things to find!")
```
again

BlueWildrose committed:
yes

Co-authored-by: SandPoot <43283559+SandPoot@users.noreply.github.com>

BlueWildrose committed:
yes

Co-authored-by: SandPoot <43283559+SandPoot@users.noreply.github.com>

DeltaFire15 committed:
minor pre-merge change

I don't feel like requesting changes for this small of a nitpick so I'll just do it myself & merge.
